pkg/util/spd_baseline.go

```diff
@@ -17,46 +17,44 @@ limitations under the License.
 package util
 
 import (
+	"encoding/json"
 	"fmt"
-	"hash/crc32"
-	"strconv"
-	"strings"
 
 	v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/util/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/kubewharf/katalyst-api/pkg/apis/workload/v1alpha1"
 	"github.com/kubewharf/katalyst-api/pkg/consts"
 )
 
-const (
-	baselineCoefficientSep = ","
-)
-
-type BaselineCoefficient []int64
+type PodMeta struct {
```

Comment:
To avoid conflict with other definitions, consider adding a prefix of spd baseline

Reply:
modified